phpBB/adm/style/acp_help_phpbb.html

```diff
@@ -54,11 +54,11 @@ <h2><i class="icon fa-bar-chart"></i>{L_SEND_STATISTICS}</h2>
 <form action="{U_COLLECT_STATS}" method="post" target="questionaire_result" id="questionnaire-form">
 	<fieldset>
 		<p class="submit-buttons">
-			<!-- BEGIN providers -->
-				<!-- BEGIN values -->
-					<input type="hidden" name="{providers.NAME}[{providers.values.KEY}]" value="{providers.values.VALUE}" />
-				<!-- END values -->
-			<!-- END providers -->
+			{% for providers in loops.providers %}
```

Comment:
`loops.` is necessary?

Reply:
No, but it's IMHO correct according to twig :P

Reply:
We are used without it, done in some PR of mine. :P

Reply:
https://github.com/phpbb/phpbb/pull/5560/files#diff-a2839ab7151761dc17f76d5b289b5005R3